scripts/install/README.md

```diff
@@ -50,7 +50,7 @@ The installation script requires some arguments:
 |oidc_secret | Given secret ID|
```

Comment:
does the documentation not also have to be change to "oidc_clientsecret"?

Reply:
Aangepast